widgets/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "scripts": {
+    "dev": "npm run build -- --sourcemap=inline --watch",
+    "build": "../node_modules/.bin/esbuild js/index.tsx --minify --format=esm --bundle --outdir=src/qsharp_widgets/static"
```

Comment:
Change to `"build": "node ../node_modules/etc.` (note the 'node' prefix on the command)

Reply:
Actually, should be `"build": "npx esbuild js/index.tsx --minify --format=esm --bundle --outdir=src/qsharp_widgets/static"`